src/quo2/README.md

```diff
@@ -69,6 +69,38 @@ In the image above we can see the properties are `Type`, `State`, `Size`,
   ...)
 ```
 
+### Handling Sizes
+For sizes there is a slightly special case. In the designs, sizes are referred to as integers. 
+
+To avoid having the codebase littered with magic numbers we instead have a keyword convention 
+to use in components to map these keywords with their sizes. 
+
+The convention is `:sizes/s-<number>`, e.g size `20` is `:size/s-20`
```

Comment:
Probably a typo, `:sizes/s-<number>` => `:size/s-<number>`

Reply:
Yep a typo, thought i fixed before. Will adjust!

Reply:
it's a little bit confusing, like sizes s,m,l,..etc

Reply:
Yeah I 100% agree! @ulisesmac too!

Afaik the only reason for the '/' is that it becomes a qualified keyword but I don't know what advantage this actually brings.

@cammellos, @ilmotta can we change to that or is there some advantage that the qualified keyword brings?

Reply:
Is the question about `:size-20` vs `:size/20`?

Actually, `:size/20` is considered an invalid keyword in Clojure and ClojureScript, just learned now, but `:size/x20` is not. Try it out `(namespace :size/20)` explodes. My only guess is because the *name* part of the keyword is just a number. So much so that clj-kondo doesn't even recognize the argument passed to `namespace` as valid.

I guess we have a decision then

Reply:
> Afaik the only reason for the '/' is that it becomes a qualified keyword but I don't know what advantage this actually brings.

@J-Son89, here are a characteristics about qualified keywords that can be beneficial (depends on the context). I'm sure there are others.

1. Qualified keywords possess a semantic meaning that precisely specifies the `namespace` and `name` part of the keyword. In other programming languages, teams have to come up with conventions to give this meaning and to implement how to parse them. Useful in some cases.

> The goal for qualified keywords has always been to provide an extra dimension of information for keys to reduce the chances of collision in the “information” realm. Instead of the nominal association of :id to a value, they let you be far more specific and define functions that operate exactly on “your” information.
>
> -- https://clojureverse.org/t/dont-quite-understand-rules-for-namespacing-keywords/7434

2. Subjective maybe, but the qualification stands out more than a simple dash. `:chat/cooldown-enabled?` I don't have to think for a split second to identify the value and the category/namespace.

In the end to me, qualifications are all about creating meaningful categories in the system, just like we did with translations and icons. Do we need this for sizes? I don't know.

I also dislike `:size/20` or `:size-20` because it's more like a glorified magic value due to the hardcoded 20 in the name. If a designer changes 20 to 24 we would still need to change the magic value in numerous instances in the repo, which to me misses the whole point. But I digress, I'm cool with whatever we decide here.

Reply:
> I also dislike :size/20 or :size-20 because it's more like a glorified magic value

Yes, I think at the end it's the same as `20`, I see almost the same in:
```clojure
{:foo "bar"
 :size 20}
```

than
```clojure
{:foo  "bar"
 :size :size-20}
```


But it's ok, I prefer `:size-20` over `:size/s-20` :+1: 


Thank you so much @J-Son89 for documenting this! :)  :+1:  :100:

Reply:
Done, and in this pr I also changed the uses in the codebase. I think a search and replace should be safe here 👍